src/query/execute_schema.c

```diff
@@ -14971,9 +15016,17 @@ do_recreate_saved_indexes (MOP classmop, SM_CONSTRAINT_INFO * index_save_info)
     {
       if (SM_IS_CONSTRAINT_INDEX_FAMILY ((SM_CONSTRAINT_TYPE) saved->constraint_type))
 	{
+	  SM_INDEX_STATUS old_status = SM_NORMAL_INDEX;
+
+	  error = sm_get_constr_index_status (classmop, saved->name, &old_status);
```

Comment:
same issue here. 
please look at the caller, `do_redistribute_partitions_data`.